src/openforms/logging/tests/test_events.py

```diff
@@ -33,3 +44,61 @@ def test_submission_start(self):
 
             log = TimelineLogProxy.objects.last()
             self.assertEqual(_("Anonymous user"), log.fmt_user)
+
+    def test_submission_logic_evaluated(self):
+
+        json_logic_trigger = {
+            ">": [{"date": {"var": "birthdate"}}, {"date": "2022-06-20"}]
+        }
+
+        json_logic_trigger_2 = {"==": [{"var": "firstname"}, "bar"]}
+
+        actions = [
+            {
+                "component": "firstname",
+                "formStep": "",
+                "action": {
+                    "type": "property",
+                    "property": {"value": "disabled", "type": "bool"},
+                    "state": True,
+                },
+            }
+        ]
+
+        actions_2 = [
+            {
+                "component": "",
+                "formStep": "",
+                "action": {
+                    "type": "disable-next",
+                    "property": {"type": "", "value": ""},
+                    "value": "",
+                    "state": "",
+                },
+            }
+        ]
+
+        actual = self.submission.get_merged_data()
+        expected = {"firstname": "foo", "birthdate": "2022-06-20"}
+        rule = FormLogic(
+            form=self.submission.form,
+            json_logic_trigger=json_logic_trigger,
+            actions=actions,
+        )
+        rule_2 = FormLogic(
+            form=self.submission.form,
+            json_logic_trigger=json_logic_trigger_2,
+            actions=actions_2,
+        )
+
+        logevent.submission_logic_evaluated(
+            self.submission,
+            [{"rule": rule, "trigger": True}, {"rule": rule_2, "trigger": False}],
+        )
+        logs = self.submission.logs.all()
```

Comment:
```suggestion
        logevent.submission_logic_evaluated(
            self.submission,
            [{"rule": rule, "trigger": True}, {"rule": rule_2, "trigger": False}],
        )
        
        logs = self.submission.logs.all()
```
a visual separation between "call" and "assertions" makes it easier to understand what is being tested